env/mock_env.cc

```diff
@@ -780,4 +780,14 @@ void MockEnv::FakeSleepForMicroseconds(int64_t micros) {
   fake_sleep_micros_.fetch_add(micros);
 }
 
+#ifndef ROCKSDB_LITE
+// This is to maintain the bahavior before swithcing from InMemoryEnv to MockEnv
+Env* NewMemEnv(Env* base_env) { return new MockEnv(base_env); }
```

Comment:
oh I just realized there is nothing like `NewMockEnv()`, so no alternative for users. I guess let's leave as-is since it doesn't seem worthwhile to deprecate just to rename it.